staff/acct/sorry/sorry

```diff
@@ -32,6 +36,15 @@ if [ -z "$(getent passwd "$sorriedUser")" ]; then
 fi
 
 sorryFile=$2
+
+if [[ ! -f "$sorryFile" ]]; then
+    sorryFile="$REASONPATH$sorryFile"
```

Comment:
for clarity I'd move the slash from $REASONPATH to here, i.e. $REASONPATH/$sorryFile